tests/features/event.feature

```diff
@@ -191,6 +197,15 @@ Feature: Event content creation
     Then I fill in "Name" with "Name of the event partner" in the "Event partner" region
     And I fill in "Use existing media" with "Euro with miniature figurines" in the "Event partner" region
     And I fill in "Website" with "http://eventpartner.com" in the "Event partner" region
+    # Event contact field group.
+    Then I fill in "Name" with "Name of the event contact" in the "Event contact" region
+    And I select the radio button "General contact"
+    And I select "Belgium" from "Country" in the "Event contact" region
+    And I wait for AJAX to finish
+    And I fill in "Street address" with "Rue belliard 28" in the "Event contact" region
+    And I fill in "Postal code" with "1000" in the "Event contact" region
+    And I fill in "City" with "Brussels" in the "Event contact" region
+    And I fill in "Email" with "test@example.com" in the "Event contact" region
```

Comment:
Should you not check the values on the view page the same way you did with the Partner inormation?